engine/src/main/java/org/terasology/rendering/opengl/PBO.java

```diff
@@ -30,7 +30,8 @@
  * Created by manu on 15.03.2015.
  */
 public class PBO {
-    private int pboId;
+    private ByteBuffer resultBuffer = null;
```

Comment:
bump: also this hasn't been addressed in the latest commit. Please revert to original version.